### PR TITLE
Feature/feature icon medal

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

File: src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java [hunk 1]
```diff
@@ -292,19 +292,19 @@ public void createForum() throws DatabaseAccessException, DocumentException {
...
                 willDoNothing().given(documentAdapter)
-                    .updateDocumentFields(anyString(), anyString(), any(FieldValue.class), any(WriteBatch.class));
+                    .updateDocumentFields(any(WriteBatch.class), anyString(), anyString(), any(FieldValue.class));
                 mockAddForumToTag();
 
                 dao.updateTags(groupName, forumName, tags, tags);
                 verify(documentAdapter)
-                    .updateDocumentFields(eq(forumPath), eq("tags"), eq(FieldValue.arrayRemove(tags.toArray())),
-                        same(batch));
+                    .updateDocumentFields(same(batch), eq(forumPath), eq("tags"),
```
Comment (santidrj) [inline src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java:300]: Codacy Issue found: [The String "tags" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoImpl.java  1
         

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoImpl.java  3
- src/main/java/org/pesmypetcare/webservice/entity/medalmanager/UserMedalEntity.java  1
- src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852&bid=18499935)

File: src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java [hunk 2]
```diff
@@ -51,6 +51,7 @@
     private static final String UPDATE_FAILED_CODE = "update-failed";
     private static final String FIELD_LIKED_BY = "likedBy";
     private static final String FCM = "FCM";
+    private static final String NOTIFICATIONS_FIELD = "notification-tokens";
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java:54]: Codacy Issue found: [The String "notification-tokens" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852)

**Review by AlbertPG — APPROVED** [review]
Okey

**Review by marclll — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Coverage increased per file
===========================
+ src/main/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoImpl.java  1
         

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoImpl.java  3
- src/main/java/org/pesmypetcare/webservice/entity/medalmanager/UserMedalEntity.java  1
- src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852&bid=18499935)

**Pull request merged**